README.md

```diff
@@ -55,6 +55,17 @@ For example:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element, { displayMode: true });
 ```
 
+Note that the `throwOnError` option only deals with how TeX related parsing errors are handled.
```

Comment:
After the discussion here I see I don't have the required understanding to make this change in the docs so I've removed it from this PR in this commit: https://github.com/Khan/KaTeX/pull/540/commits/42e21c2232e55dc360578244feee280a00510ae4